src/app/core/admin/admin-list-manager/admin-list-manager.component.html

```diff
@@ -19,6 +19,7 @@
             <mat-form-field appearance="fill">
               <mat-label>
                 <app-entity-field-label
+                  [customFields]="activeFields"
```

Comment:
I am confused why "activeFields" would be customFields here. Checking AdminEntityList component, what is input here (from the list config "columns" is actually "additionalFields" rather than activeFields, I think?

I would suggest we clean up this old code. The naming and structure used in the other PR (https://github.com/Aam-Digital/ndb-core/pull/3135/files#diff-6620db6be9df8a567daeb65d7ef6929ad0ab0c19621cb48def90ae2a3813103b) seems well structured and clearly named. Let's use this here and adapt admin-entity-list to use it's extended functionality, if we can?